catalogs/sources/gtfs/schedule/us-california-wasco-1782.json

```diff
@@ -0,0 +1,16 @@
+{
+    "mdb_source_id": 1782,
+    "data_type": "gtfs",
+    "provider": "City of Wasco",
+    "features": ["flex","flex-v2"],
+    "status": "development",
+    "location": {
+        "country_code": "US",
+        "subdivision_name": "California",
+        "municipality": "Wasco"
```

Comment:
The bounding box information is missing, so the integration tests are failing.

Reply:
Gotcha - I will use  `tools.operations.add_gtfs_schedule_source()` but in general it would be a nice feature that the calculated fields could be computed if missing ;-)